src/test_parenthetics.py

```diff
@@ -0,0 +1,20 @@
+"""Test proper parenthetics."""
+
+import pytest
+
+
+EXAMPLE_PARENS = [
+    ['()()()', 0],
```

Comment:
I would've liked to see a couple tests of balanced strings with nested parentheses